rrmngmnt/package_manager.py

```diff
@@ -81,34 +107,59 @@ def install(self, package):
                 "Install package %s on host %s", package, self.host
             )
             return self._run_command_on_host(cmd)
-        self.logger.info(
+        self. logger.info(
             "Package %s already exist on host %s", package, self.host
         )
         return True
 
-    def remove(self, package):
+    def remove(self, package, pattern=False):
         """
-        Remove package from host
+        Remove package from host, or packages which match pattern if pattern
+        is set to True
 
-        :param package: name of package
+        :param package: name of package or regular expression pattern
         :type package: str
-        :return: True, if package removal success, otherwise False
+        :param pattern: If True package name is pattern
+        :return: True, if package(s) removal success, otherwise False
         :rtype: bool
         :raise: NotImplementedError
         """
         if not self.remove_command_d:
-            raise NotImplementedError("There is no 'remove' command defined.")
+            raise NotImplementedError("there is no 'remove' command defined.")
```

Comment:
Why did you lowercase the fist letter? Sentence should start with capital letter ...